### PR TITLE
appcenter-codepush-release-react-native 0.0.1

### PR DESCRIPTION
**Pull request by adrianchinghc**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by viktorbenei** [comment]
Thanks for the PR @adrianchinghc ! We'll review it ASAP! :)

P.S.: as this is a new step the first review will be more thorough, subsequent updates will be faster as we'll only have to check the diffs ;)

**Comment by adrianchinghc** [comment]
👌 👍

File: steps/appcenter-codepush-release-react-native/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    summary: AppCenter application identifier (Organization/Owner Name + App Name).
+    title: AppCenter App ID
+- api_token: $APPCENTER_API_TOKEN
+  opts:
+    description: |
+      AppCenter API Token can be obtained from [here]](https://appcenter.ms/settings/apitokens).
+      New API Token > Enter a description for the token > Select Full Access > Add New API Token
+    is_required: true
+    title: AppCenter API Token
+- env: $ENVIRONMENT
```
Comment (godrei) [inline steps/appcenter-codepush-release-react-native/0.0.1/step.yml]: We add default value to a step input, if we can specify a working initial value for the input. Like xcode-archive step's project_path input, the BITRISE_PROJECT_PATH env is always defined for ios projects (if our scanner generated the initial config on the Add new app page).

File: steps/appcenter-codepush-release-react-native/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  opts:
+    description: |
+      Choices for environments [here](https://github.com/Microsoft/appcenter-cli/blob/61bd72c1dcabeb876ee14fcaa316fd6b367b49a6/src/commands/codepush/lib/environment.ts#L18)
+    is_required: true
+    title: Environment
+- options: $OPTIONS
```
Comment (godrei) [inline steps/appcenter-codepush-release-react-native/0.0.1/step.yml]: please remove the default value

**Comment by adrianchinghc** [comment]
@godrei Updated.

**Comment by godrei** [comment]
@adrianchinghc thanks for the update! It looks great to me.

**Comment by adrianchinghc** [comment]
Awesome! Thanks @godrei

**Pull request merged**